cmd/minikube/cmd/config/util.go

```diff
@@ -88,11 +88,67 @@ func GetClientType() machine.ClientType {
 }
 
 func EnableOrDisableAddon(name string, val string) error {
+
 	enable, err := strconv.ParseBool(val)
 	if err != nil {
 		errors.Wrapf(err, "error attempted to parse enabled/disable value addon %s", name)
 	}
 
+	// allows for additional prompting of information when enabling addons
+	if enable {
+		switch name {
+		case "registry-creds":
+			posResponses := []string{"yes", "y"}
+			negResponses := []string{"no", "n"}
+
+			enableAWSECR := AskForYesNoConfirmation("\nDo you want to enable AWS Elastic Container Registry?", posResponses, negResponses)
+			if enableAWSECR {
+				awsAccessID := AskForStaticValue("-- Enter AWS Access Key ID: ")
+				awsAccessKey := AskForStaticValue("-- Enter AWS Secret Access Key: ")
+				awsRegion := AskForStaticValue("-- Enter AWS Region: ")
+				awsAccount := AskForStaticValue("-- Enter 12 digit AWS Account ID: ")
+
+				cluster.CreateSecret(
```

Comment:
I believe that for GCR/ECR to work in registry-creds current state, both secrets will need to be created for the pod to start properly.  This was the case in my experience at least.  Maybe empty values can be used for secrets specify which mode is enabled? We can also make changes to registry-creds itself.